ecl/eclagent/eclagent.cpp

```diff
@@ -3184,7 +3184,7 @@ char *EclAgent::getFilePart(const char *lfn, bool create)
         expandLogicalName(full_lfn, lfn);
 
         StringBuffer physical;
-        makePhysicalPartName(full_lfn.str(), 1, 1, physical, false);//MORE: What do we do if local ?
+        makePhysicalPartName(full_lfn.str(), 1, 1, physical, 0, DFD_OSdefault, nullptr, false);//MORE: What do we do if local ?
```

Comment:
similar question about never creating in a subdir.

Reply:
1 part files will not be created/published with the subdirperpart flag